MoneySplitter.Win10/ViewModels/RegisterViewModel.cs

```diff
@@ -5,10 +5,17 @@
 
 namespace MoneySplitter.Win10.ViewModels
 {
-    public  class RegisterViewModel : Screen
+    public class RegisterViewModel : Screen
     {
         private readonly INavigationManager _navigationManager;
         private readonly IMembershipService _membershipService;
+        private readonly IFilePickerService _filePickerService;
+
+        private const string DEFEALT_TEXT_LABEL_AVATAR = "Browse avatar image";
```

Comment:
Just paste this texts to code. We will change it to extensions.Resource="" later